jaeger_client/thrift.py

```diff
@@ -121,59 +69,50 @@ def timestamp_micros(ts):
     return long(ts * 1000000)
 
 
-def make_zipkin_spans(spans):
-    zipkin_spans = []
+def make_tags(tags):
+    # TODO extend to support non-string tag values
+    return [
+        make_string_tag(key=k, value=str(v), max_length=1024)
```

Comment:
MAX_TAG_VALUE_LENGTH?

Reply:
well if that then better to use the configurable value, which will need to be pushed through. I can do it.